### ISSUE TITLE
Kohya not seeing training images

### ISSUE THREAD

**Issue by WTW84** [kohya-ss/sd-scripts#2076]
Hi, I'm using the following script for training:

@echo off

call venv\Scripts\activate.bat

:: Set paths WITHOUT trailing spaces
set TRAIN_DATA_DIR=K:\img\1_images
set OUTPUT_DIR=K:\output
set LOGFILE=training_log.txt

(
echo Starting training...
dir "%TRAIN_DATA_DIR%" /A-D /B
timeout /t 3

:: Run training with proper quoting
python train_network.py ^
 --pretrained_model_name_or_path="K:\SD\stable-diffusion-webui\models\Stable-diffusion\epicphotogasm_v1.safetensors" ^
 --train_data_dir=%TRAIN_DATA_DIR% ^
 --output_dir=%OUTPUT_DIR% ^
 --resolution=512 ^
 --network_alpha=16 ^
 --network_dim=128 ^
 --learning_rate=1e-5 ^
 --max_train_steps=6000 ^
 --save_every_n_steps=1000 ^
 --save_model_as=safetensors ^
 --output_name=okskface_lora ^
 --cache_latents ^
 --train_batch_size=2 ^
 --gradient_checkpointing ^
 --mixed_precision=fp16 ^
 --max_data_loader_n_workers=4 ^
 --xformers ^
 --shuffle_caption ^
 --debug_dataset
) > %LOGFILE% 2>&1

pause

And in the log I get this:

rich is not installed, using basic logging
prepare tokenizer
Using DreamBooth method.
prepare images.
0 train images with repeating.
0 reg images.
no regularization images / ж­Је‰‡еЊ–з”»еѓЏгЃЊи¦‹гЃ¤гЃ‹г‚ЉгЃѕгЃ›г‚“гЃ§гЃ—гЃџ
[Dataset 0]
  batch_size: 2
  resolution: (512, 512)
  enable_bucket: False
  network_multiplier: 1.0

[Dataset 0]
loading image sizes.

0it [00:00, ?it/s]
0it [00:00, ?it/s]prepare dataset
Total dataset length (steps) / гѓ‡гѓјг‚їг‚»гѓѓгѓ€гЃ®й•·гЃ•пј€г‚№гѓ†гѓѓгѓ—ж•°пј‰: 0
`S` for next step, `E` for next epoch no. , Escape for exit. / Sг‚­гѓјгЃ§ж¬ЎгЃ®г‚№гѓ†гѓѓгѓ—гЂЃEг‚­гѓјгЃ§ж¬ЎгЃ®г‚Ёгѓќгѓѓг‚ЇгЂЃEscг‚­гѓјгЃ§дё­ж–­гЂЃзµ‚дє†гЃ—гЃѕгЃ™

For some reason, it is not seeing my images, even though the path is correct and the images are in the correct format. Other scripts have no trouble accessing them. I have consulted deepseek and chatgpt on this issue, but they haven't been very helpful. Any advice would be appreciated.

**Comment by WTW84** [comment]
That helped, thank you
